static/styles/settings.css

```diff
@@ -2040,3 +2029,18 @@ input[type="checkbox"] {
         margin-top: 10px;
     }
 }
+
+.settings_panel_list_header {
+    position: relative;
+
+    h3 {
+        display: inline-block;
+    }
+
+    input.search {
+        float: right;
+        font-size: 1em;
+        max-width: 160px;
```

Comment:
I changed this 160px before merging because I'd done the same in the subscriptions PR (where you'd done 206px=>130px, and I decided that was a bit too small; forgot to note it), and we'll hopefully eventually deduplicate CSS between the two.